core/src/main/java/com/graphhopper/routing/util/BikeFlagEncoder.java

```diff
@@ -86,6 +86,7 @@ boolean isPushingSection( OSMWay way )
         String trackType = way.getTag("tracktype");
         return way.hasTag("highway", pushingSections)
                 || way.hasTag("railway", "platform")
+                || way.hasTag("bicycle", "dismount")
```

Comment:
Thank you. This is a good optimisation. I have updated the PR with this included.